dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -125,7 +125,7 @@ xPoweredBy.name=X-Powered-By
 xPoweredBy.desc=Alternative string for the X-Powered-By header setting. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.xpoweredby. There is no default value for this property. If the property is not set, the value of the X-Powered-By header is set to Servlet/<servlet spec version>, as defined by the Servlet specification.
 
 disableXPoweredBy.name=Disable the X-Powered-By header
-disableXPoweredBy.desc=Disable setting the X-Powered-By header. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.disablexpoweredby.
+disableXPoweredBy.desc=Disable setting the X-Powered-By header. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.disablexpoweredby. The default is true for servlet-5.0 and newer Servlet features.
```

Comment:
**change**
 The default is true for servlet-5.0 and newer Servlet features.
**to**
 For servlet-5.0 and newer Servlet features, the default value is true .

Reply:
Thanks @dmuelle  I made the requested change!